recipes/rich/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "rich" %}
+{% set version = "0.8.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c783f1c99cc709992e6a1cf9f0d04fc27c205a8920a87d1e24ced0aab49be26f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<36]
+
+requirements:
+  host:
+    - python
+    - pip
+    - poetry
+  run:
+    - python
```

Comment:
I think `pygments` and `pprintpp` are required dependencies

Reply:
Ah true. However it seems it has even more dependencies: https://github.com/willmcgugan/rich/blob/master/pyproject.toml#L26-L31

However `dataclasses` and `typing-extensions` only seem to be necessary for Python 3.6.

How could this be represented in the recipe? I left them out for now in the dependency list.

Reply:
I think it's only `dataclasses` that is a 3.6 requirement according to the pyproject.toml. That said, I think what needs to happen is we drop the `noarch: python` and then add the `# [py36]` tag to `dataclasses`